src/table.ts

```diff
@@ -951,36 +825,13 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
             userStream.end();
             return;
           }
-          numConsecutiveErrors++;
-          numRequestsMade++;
-          if (
-            numConsecutiveErrors <= maxRetries &&
-            (RETRYABLE_STATUS_CODES.has(error.code) || isRstStreamError(error))
-          ) {
-            const backOffSettings =
-              options.gaxOptions?.retry?.backoffSettings ||
-              DEFAULT_BACKOFF_SETTINGS;
-            const nextRetryDelay = getNextDelay(
-              numConsecutiveErrors,
-              backOffSettings
-            );
-            retryTimer = setTimeout(makeNewRequest, nextRetryDelay);
-          } else {
-            userStream.emit('error', error);
-          }
-        })
```

Comment:
This here was the old custom retry logic. Retries are done in gax now so this should be removed.